oxenss/rpc/client_rpc_endpoints.cpp

```diff
@@ -521,6 +521,43 @@ bt_value delete_msgs::to_bt() const {
     return ret;
 }
 
+
+template <typename Dict>
+static void load(revoke_subkey& rs, Dict& d) {
+    auto [pubkey, pubkey_ed25519, revoke_subkey, signature] =
+            load_fields<Str, SV, SV, SV>(
+                    d, "pubkey", "pubkey_ed25519", "revoke_subkey", "signature");
+    load_pk_signature(rs, d, pubkey, pubkey_ed25519, signature);
+    require("revoke_subkey", revoke_subkey);
+    const auto& sk = *revoke_subkey;
+    if constexpr (std::is_same_v<json, Dict>) {
+        if (oxenc::is_base64(sk) && (sk.size() == 43 || (sk.size() == 44 && sk.back() == '=')))
+            oxenc::from_base64(sk.begin(), sk.end(), rs.revoke_subkey.begin());
+        else if (oxenc::is_hex(sk) && sk.size() == 64)
+            oxenc::from_hex(sk.begin(), sk.end(), rs.revoke_subkey.begin());
+        else
+            throw parse_error{
+                    "invalid subkey: expected base64 or hex-encoded 32-byte subkey index"};
```

Comment:
```suggestion
                    "invalid subkey: expected base64 or hex-encoded 32-byte subkey tag"};
```
(to keep it consistent with the description elsewhere)